tests/e2e/utils/requestHandlers/tokens/CheMultiuserTokenHandler.ts

```diff
@@ -12,22 +12,14 @@ import querystring from 'querystring';
 import { injectable } from 'inversify';
 import { TestConstants } from '../../../TestConstants';
 import { ITokenHandler } from './ITokenHandler';
+import { KeycloackUrlHandler } from '../../KeycloackUrlHandler';
 
 @injectable()
 export class CheMultiuserTokenHandler implements ITokenHandler {
     async get(): Promise<string> {
-        let params = {};
+        const keycloakUrl = KeycloackUrlHandler.getTokenEndpointUrl();
 
-        let keycloakUrl = this.handleTrailingSlash(TestConstants.TS_SELENIUM_BASE_URL);
-        const keycloakAuthSuffix = 'auth/realms/che/protocol/openid-connect/token';
-
-        if (TestConstants.TS_SELENIUM_SINGLE_HOST) {
-            keycloakUrl = keycloakUrl + keycloakAuthSuffix;
-        } else {
-            keycloakUrl = keycloakUrl.replace('che', 'keycloak') + keycloakAuthSuffix;
```

Comment:
This case has been completely omitted with the removal

Reply:
No, it is handling in the "getBaseKeycloakUrl".
I would say that it was completely simplified.

Reply:
I see it now, sorry